core/templates/pages/exploration-editor-page/modal-templates/welcome-modal.component.ts

```diff
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Component for welcome modal.
  */
+
```

Comment:
Unneeded changne.

Reply:
Reverted, but I think we are doing this for all the migrated files so I did this here as well.